src/polyfills/custom-elements.js

```diff
@@ -958,7 +958,7 @@ export function install(win, ctor) {
       const {Reflect} = win;
 
       // "Construct" ctor using ES5 idioms
-      const instance = Object.create(ctor.prototype);
+      const instance = /** @type {!Function} */ (Object.create(ctor.prototype));
```

Comment:
Why is this the right type?

Reply:
I'm not really sure, but closure doesn't like the `Function.call.call()` below without it.

Reply:
Maybe add a comment?

Reply:
Done.